lxd/network/driver_bridge.go

```diff
@@ -212,71 +268,368 @@ func (n *bridge) Validate(config map[string]string) error {
+		//  condition: IPv6 DHCP
+		//  defaultdesc: `1h`
+		//  shortdesc: When to expire DHCP leases
+		"ipv6.dhcp.expiry": validate.IsAny,
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=ipv6.dhcp.stateful)
+		//
+		// ---
+		//  type: bool
+		//  condition: IPv6 DHCP
+		//  defaultdesc: `false`
+		//  shortdesc: Whether to allocate IPv6 addresses using DHCP
+		"ipv6.dhcp.stateful": validate.Optional(validate.IsBool),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=ipv6.dhcp.ranges)
+		// Specify a comma-separated list of IPv6 ranges in FIRST-LAST format.
+		// ---
+		//  type: string
+		//  condition: IPv6 stateful DHCP
+		//  defaultdesc: all addresses
+		//  shortdesc: IPv6 ranges to use for DHCP
+		"ipv6.dhcp.ranges": validate.Optional(validate.IsListOf(validate.IsNetworkRangeV6)),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=ipv6.routes)
+		// Specify a comma-separated list of IPv6 CIDR subnets.
+		// ---
+		//  type: string
+		//  condition: IPv6 address
+		//  shortdesc: Additional IPv6 CIDR subnets to route to the bridge
+		"ipv6.routes": validate.Optional(validate.IsListOf(validate.IsNetworkV6)),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=ipv6.routing)
+		//
+		// ---
+		//  type: bool
+		//  condition: IPv6 address
+		//  shortdesc: `true`
+		//  shortdesc: Whether to route IPv6 traffic in and out of the bridge
+		"ipv6.routing": validate.Optional(validate.IsBool),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=ipv6.ovn.ranges)
+		// Specify a comma-separated list of IPv6 ranges in FIRST-LAST format.
+		// ---
+		//  type: string
+		//  shortdesc: IPv6 ranges to use for child OVN network routers
+		"ipv6.ovn.ranges": validate.Optional(validate.IsListOf(validate.IsNetworkRangeV6)),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=dns.domain)
+		//
+		// ---
+		//  type: string
+		//  defaultdesc: `lxd`
+		//  shortdesc: Domain to advertise to DHCP clients and use for DNS resolution
+		"dns.domain": validate.IsAny,
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=dns.mode)
+		// Possible values are `none` for no DNS record, `managed` for LXD-generated static records, and `dynamic` for client-generated records.
+		// ---
+		//  type: string
+		//  defaultdesc: `managed`
+		//  shortdesc: DNS registration mode
+		"dns.mode": validate.Optional(validate.IsOneOf("dynamic", "managed", "none")),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=dns.search)
+		// Specify a comma-separated list of domains.
+		// ---
+		//  type: string
+		//  defaultdesc: `dns.domain` value
+		//  shortdesc: Full domain search list
+		"dns.search": validate.IsAny,
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=dns.zone.forward)
+		// Specify a comma-separated list of DNS zone names.
+		// ---
+		//  type: string
+		//  defaultdesc: `managed`
```

Comment:
This default value doesn't make sense. It might be ``[`managed`]`` (if `managed` is the DNS zone name)?

Reply:
After looking in the codebase, I don't see any default value for `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6`. Letting them empty is valid.